lib/core/sync_status_title.dart

```diff
@@ -13,7 +13,7 @@ String syncStatusTitle(SyncStatus syncStatus, SyncStatusDisplayMode syncStatusDi
       // Get ETA with placeholder while gathering data
       String eta = syncStatus.getFormattedEtaWithPlaceholder() ?? '';
 
-      if (eta.isEmpty) {
+      if (eta.isEmpty || eta == '--:--') {
```

Comment:
revert this change please, it should show the eta even if still empty as long as the user has this settings on